### PR TITLE
Remove obsolete Profile page and related assets

### PR DESCRIPTION
**Pull request by rhamilto**
Fixes https://jira.coreos.com/browse/CONSOLE-999

### CODE REVIEW

**Comment by spadgett** [comment]
Hey @rhamilto, before we remove this completely, I'd like to investigate if we can still support downloading a kubeconfig file in OpenShift. I'd need to dig into how it works, though.

If we go ahead with the change, there is some backend code we'd remove as well.

**Comment by spadgett** [comment]
/hold
to look at kubeconfig download

**Comment by rhamilto** [comment]
Replaced by https://github.com/openshift/console/pull/789

**Comment by spadgett** [comment]
> Replaced by #789

It looks a couple of the files are still there (profile.jsx, kubectl-config.jsx)

**Comment by rhamilto** [comment]
Rebased.

**Comment by alecmerdler** [comment]
@rhamilto Could you please remove the Go server's HTTP route handlers as well?

**Comment by alecmerdler** [comment]
@rhamilto https://github.com/openshift/console/blob/master/server/server.go#L164

**Comment by rhamilto** [comment]
> @rhamilto https://github.com/openshift/console/blob/master/server/server.go#L164

Thanks!  Updated.

File: frontend/public/components/client-tokens.jsx [hunk 1]
```diff
@@ -1,99 +0,0 @@
...
-    this._getClients = this._getClients.bind(this);
-  }
-
-  componentDidMount() {
-    super.componentDidMount();
-    this._getClients();
-  }
-
-  _getClients() {
-    coFetchJSON('api/tectonic/clients')
```
Comment (alecmerdler) [inline frontend/public/components/client-tokens.jsx]: Remove server route.

File: frontend/public/components/client-tokens.jsx [hunk 2]
```diff
@@ -1,99 +0,0 @@
...
-const RevokeToken = (id, onTokenRevocation) => ({
-  label: 'Revoke Access...',
-  callback: () => confirmModal({
-    title: 'Confirm Revocation',
-    message: 'Revoking this client\'s access token will immediately remove it. Once removed, you cannot recover the token. If this is the last client authorized, you will loose access to the Console after submitting it.',
-    btnText: 'Revoke Access',
-    executeFn: () => {
-      const data = new FormData();
-      data.append('clientId', id);
-      const promise = coFetch('api/tectonic/revoke-token', {
```
Comment (alecmerdler) [inline frontend/public/components/client-tokens.jsx]: Remove server route.

File: frontend/public/components/modals/kubectl-config.jsx [hunk 3]
```diff
@@ -1,161 +0,0 @@
...
-import {saveAs} from 'file-saver';
-
-import { k8sVersion } from '../../module/status';
-import { createModalLauncher, ModalTitle, ModalBody, ModalFooter, ModalSubmitFooter } from '../factory/modal';
-import { PromiseComponent } from '../utils';
-import {coFetch} from '../../co-fetch';
-
-
-const getVerificationCode = () => {
-  window.open('api/tectonic/kubectl/code');
```
Comment (alecmerdler) [inline frontend/public/components/modals/kubectl-config.jsx]: Remove server route.

File: frontend/public/components/modals/kubectl-config.jsx [hunk 4]
```diff
@@ -1,161 +0,0 @@
...
-};
-
-const getConfiguration = code => {
-  return coFetch('api/tectonic/kubectl/config', {
```
Comment (alecmerdler) [inline frontend/public/components/modals/kubectl-config.jsx]: Remove server route.

**Comment by rhamilto** [comment]
Now with bunches of Go deletions.  Definitely needs a thorough review since I'm not a Go coder.

**Comment by rhamilto** [comment]
Note links to the profile page also exist in #813

**Comment by alecmerdler** [comment]
Fix up the unused Go vars that are causing the backend build to fail.

**Comment by rhamilto** [comment]
jenkins rebuild

**Comment by alecmerdler** [comment]
@rhamilto Ready for your review.

**Comment by rhamilto** [comment]
lgtm.  @spadgett?

**Comment by alecmerdler** [comment]
@spadgett Is this good to merge?

**Comment by spadgett** [comment]
Sorry I left this for so long

**Comment by alecmerdler** [comment]
We still haven't figured out a way to re-implement `kubeconfig` download, right?

**Comment by spadgett** [comment]
There's a token request link that the CLI will print out for some identity providers like Github if you use `oc login`. You go there, login, and it will show you your token with a `oc login --token=<your-token>` command you can copy.

It's easy enough to implement in the console if we are willing to take HTTP-only off of the cookie. It'd need an exception for 4.0, though.

**Pull request merged**